CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+### Fixed
+
+- #3601, Fix jwt error return status `400` for invalid role - @taimoorzaeem
```

Comment:
I'll try changing to this format from now on, to reduce editing effort on release notes ([ref](https://github.com/PostgREST/postgrest/pull/4020#issuecomment-2868219975)).

```suggestion
- Fix jwt error returning HTTP status `400` for invalid role by @taimoorzaeem in #3601
```